llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -2714,7 +2714,7 @@ def int_amdgcn_ds_sub_gs_reg_rtn :
             [ImmArg<ArgIndex<1>>, IntrHasSideEffects, IntrWillReturn, IntrNoCallback, IntrNoFree],
             "", [SDNPMemOperand]>;
 
-def int_amdgcn_ds_bvh_stack_rtn :
+class IntDSBVHStackRtn :
```

Comment:
Could make this class parameterized on `vdst_ty` and `data1_ty` so it can be used for all three intrinsics?